src/c++/perf_analyzer/genai-perf/genai_perf/metrics/metrics.py

```diff
@@ -26,42 +26,33 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from dataclasses import dataclass
 from enum import Enum, auto
 from typing import List
 
 
 class ResponseFormat(Enum):
     OPENAI_CHAT_COMPLETIONS = auto()
     OPENAI_COMPLETIONS = auto()
+    OPENAI_EMBEDDINGS = auto()
     TRITON = auto()
 
 
+@dataclass
+class Metric:
```

Comment:
An alternative thought, we could go another way and change Metrics to one of the following:
1. Measurements
2. GatheredMeasurements
3. ProfiledMeasurements

Reply:
That's a valid concern. I want to change the `Metric` to other name if I could (since the change is much simpler) but can't think of any better name than `Metric` 😅 I will think about what we can change `Metrics` name.

Reply:
Final decision was Metric -> MetricMetaData